.github/workflows/test.yml

```diff
@@ -7,22 +7,25 @@ jobs:
     name: Tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2 # step 1. checkout repository
-      - uses: actions/setup-node@v1 # step 2. install node
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install node
+        uses: actions/setup-node@v2
         with:
-          node-version: "14.x"
+          node-version: "15.x"
```

Comment:
Node's LTS (Long Term Support is 14), should we be moving to 15?
![image](https://user-images.githubusercontent.com/14998054/108291891-0aafec80-7161-11eb-88e7-9df35bd0874f.png)

Reply:
Node 15 is the first version that comes with **npm** 7, so that's why I moved up to 15. That said, Node 15 will die in a few months from now to birth Node 16 (which will again be LTS). I don't anticipate that we'll run into any problems by being on the latest version. We're building something that's constantly moving forward (just like Node), so we won't get stuck on an unsupported version - just upgrade when the time comes. Moreover, we're doing pretty standard stuff so we can be pretty confident that our use cases will not be broken in future releases.

https://nodejs.org/en/about/releases/
https://www.reddit.com/r/node/comments/61x4tq/lts_or_no/

Reply:
https://blog.logrocket.com/node-js-15-whats-new-and-how-the-developer-experience-has-improved/